examples/intelliJPlugin/src/main/kotlin/com/jetbrains/compose/theme/intellij/SwingColor.kt

```diff
@@ -0,0 +1,35 @@
+package com.jetbrains.compose.theme.intellij
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.graphics.Color
+import javax.swing.UIManager
+import java.awt.Color as AWTColor
+
+interface SwingColor {
+    companion object : SwingColor by SwingColorImpl
+
+    val background: Color
+    val onBackground: Color
+}
+
+object SwingColorImpl : SwingColor {
```

Comment:
This singleton object doesn't look optimal (as almost any mutable singleton), could we instead pass color change notifications to plugin instance.

Reply:
Yeh, I agree with that, but the idea here was to do something copy-paste.
I will update it to register the listener programmatically and use the lifecycle of the Composable to unregister the listener.